_docs/guides.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Guides & Tutorials
+order: 5
+---
+
+# Guides & Tutorials
+
+* [Kamal + Hetzner = Joy](https://devscoach.com/blog/kamal-hetzner-setup) <br/> _Sam Johnson_, _12 Dec 2023_
+* [Basic guide to deploy a Rails 7 application using Kamal on Hetzner Cloud](https://joshio1.blog/basic-guide-to-deploy-a-rails-7-application-using-kamal-on-hetzner-cloud/) <br/> _Omkar Joshi_, _2 Nov 2023_
+* [How To Use Basecamp’s Kamal With AWS and GitHub](https://jetrockets.com/blog/how-to-use-basecamp-s-kamal-with-aws-and-github) <br/> _Igor Alexandrov_, _21 Sep 2023_
+* [Deploying a Rails app with Kamal](https://greg.molnar.io/blog/deploying-a-rails-app-with-kamal) <br/> _Greg Molnar_, _20 Sep 2023_
+* [Deploying NextJS app using Mrsk (Kamal)](https://medium.com/@ruslanfg/deploying-nextjs-using-mrsk-819aa9eb83ba)<br/> _Ruslan Gainutdinov_, _10 Apr 2023_
```

Comment:
Hi @igor-alexandrov - I think we should remove this last one as it predates the move from MRSK to Kamal.

Reply:
@djmb yes, and add a bunch of new articles, I will update the PR.

Reply:
@djmb I removed the guide you were referring to and added two new articles.